pkg/streamingpromql/engine_test.go

```diff
@@ -39,23 +38,21 @@ import (
 	"github.com/grafana/mimir/pkg/streamingpromql/testutils"
 	"github.com/grafana/mimir/pkg/streamingpromql/types"
 	"github.com/grafana/mimir/pkg/util/globalerror"
+	"github.com/grafana/mimir/pkg/util/promqlext"
 )
 
 func init() {
 	types.EnableManglingReturnedSlices = true
+	// To test double_exponential_smoothing and holt_winters
+	promqlext.ExtendPromQL()
 }
 
 func TestUnsupportedPromQLFeatures(t *testing.T) {
 	features := EnableAllFeatures
 
-	// Disable experimental so that parser will parse it without the updating experiemental flag
-	parser.Functions["double_exponential_smoothing"].Experimental = false
-
 	// The goal of this is not to list every conceivable expression that is unsupported, but to cover all the
 	// different cases and make sure we produce a reasonable error message when these cases are encountered.
-	unsupportedExpressions := map[string]string{
-		"double_exponential_smoothing(metric{}[1h], 1, 1)": "'double_exponential_smoothing' function",
-	}
+	unsupportedExpressions := map[string]string{}
```

Comment:
Should we place one of the other unsupported experimental functions in here?

Reply:
Should like this be good enough? 1e70c324bd48135a3c2b8dc8373bcd41861614f0